packages/ng/forms/textarea-input/textarea-input.component.ts

```diff
@@ -18,4 +18,7 @@ export class TextareaInputComponent {
 
 	@Input()
 	placeholder: string = '';
+
+	@Input()
+	rows?: number;
```

Comment:
Should we set 2 as the default value? It's the official default value.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows
```suggestion
	rows: number = 2;
```

Reply:
Good point. 

The idea was to respect the design, but the question also existed of modifying it.

Reply:
Good point.
The idea was to respect the design, but the question also existed of modifying it.

Does this make the parameter mandatory? I think not.

Reply:
No, if the parameter is not provided, the default value will be used.

Reply:
On the design side, the intention is to stay with 3 lines by default, so we'll keep it that way.